event_registration_cancel_reason/wizard/event_registration_cancel_log_reason.py

```diff
@@ -23,12 +23,13 @@ def default_get(self, var_fields):
             var_fields)
         registrations = self.env['event.registration'].browse(
             self.env.context['active_ids'])
-        event_type = registrations.mapped("event_id.type")
-        if len(event_type) != 1:
-            raise exceptions.ValidationError(
-                _("You cannot cancel registrations from events of different "
-                  "types at once."))
-        res['event_type_id'] = event_type.id
+        first_type = registrations[0].event_id.type
```

Comment:
I prefer the other formula. It can silently pass to the other window without problems, and it's harmless